wpeverest/user-registration
# UR-684 Fix- Disallow reset password for not approved users.

### All Submissions:






### Changes proposed in this Pull Request:

Previously, When admin change user status to pending or denied but still userwas allowing to reset password. In this PR we are not allowing any user who's status is not approved. 

### How to test the changes in this Pull Request:

1. Register an user then change that user status as pending.
2. Now try to reset password as user in frontend. 
3. Check whether you are allowed to reset password or not.

### Types of changes:




### Other information:




### Changelog entry

> Fix- Disallow reset password for not approved users.
